src/styles/theming/sap_fiori_3_light_dark.scss

```diff
@@ -228,6 +228,9 @@
 
   /* Grid List */
   --fdGrid_List_Item_Hover_Border_Color: var(--sapTile_Interactive_BorderColor);
+  --fdGrid_List_Item_Border_Radius: var(--sapElement_BorderCornerRadius);
+  --fdGrid_List_Item_Toolbar_Shadow: var(--sapContent_HeaderShadow);
+  --fdGrid_List_Highlight_Offset: 0.1875rem;
```

Comment:
From what I've noticed, css variables for all fiori-related themes are the same. Maybe move it to a single separate file and import it per theme?

Reply:
I thought we can left it as it is because there are no many variables, but okay, created separate files and placed it there.